apps/arena/priv/config.json

```diff
@@ -423,6 +423,7 @@
       "base_size": 110.0,
       "base_health": 440,
       "base_stamina": 3,
+      "base_cooldown_multiplier": 1,
```

Comment:
i wouldn't add this to the character, if we start to spread around this values maybe we'll lost track of what is altering the duration of cooldows, what do you think about just leave is as a player entity value and have 1 as default for everyone?

Reply:
But I think that have a hardcoded value isn't a good idea

Reply:
it would be something similar to what `bonus_damage` is doing, since this is a entity modifier that should not come from the character since it's unrelated, maybe we could refactor this in another pr to put all the hardcoded player entity data in the config file